client/src/endpoint/payment.rs

```diff
@@ -235,6 +235,24 @@ impl IntoRequest for FindPath {
     }
 }
 
+impl TryFromUri for FindPath {
+    fn try_from_wrap(wrap: &UriWrap) -> ::std::result::Result<FindPath, uri::Error> {
+        let params = wrap.params();
+        let destination_asset = AssetIdentifier::new(
+            params.get_ok("destination_asset_type")?,
+            params.get_parse("destination_asset_code").ok(),
+            params.get_parse("destination_asset_issuer").ok(),
+        )?;
+
+        Ok(FindPath {
+            source_account: params.get_parse("source_account")?,
+            destination_account: params.get_parse("destination_account")?,
+            destination_asset: destination_asset,
+            destination_amount: Amount::new(params.get_parse("destination_account")?),
```

Comment:
Technically if `Amount` implements the `FromStr` trait, you should be able to parse to it.

Reply:
Cool, makes sense. However (noob question alert 😄) I get this error:

the trait `std::convert::From<resources::amount::ParseIntError>` is not implemented for `uri::Error`

meaning the two error types are different. Here and in general, how do I deal with situations where a functions expected error type and an expressions error type differ?

Reply:
So, that would mean that we are missing a conversion for our error type for amount into the result error type.

Also, I just noticed that you are parsing the account into an amount. That can't be right.

Reply:
🤔 that is a valid point